src/main/flight/gps_rescue.c

```diff
@@ -241,7 +241,7 @@ static void rescueAttainPosition(void)
         // 20s of slow descent for switch induced sanity failures to allow time to recover
         gpsRescueAngle[AI_PITCH] = 0.0f;
         gpsRescueAngle[AI_ROLL] = 0.0f;
-        rescueThrottle = gpsRescueConfig()->throttleHover - 100;
+        rescueThrottle = positionConfig()->hover_throttle - 100;
```

Comment:
why not in camelCase?

Reply:
Yes some are done camelCase and some with underscores.
Personally I like underscores because the names are the same, and when searching code, I find every instance.
Maybe at some point we do a PR to be consistent everywhere :-) But not this one ...